src/plone/restapi/types/utils.py

```diff
@@ -129,10 +129,16 @@ def get_jsonschema_for_fti(fti, context, request, excluded_fields=None):
     if excluded_fields is None:
         excluded_fields = []
 
-    schema = fti.lookupSchema()
-    additional_schemata = tuple(getAdditionalSchemata(portal_type=fti.id))
-
-    fieldsets = get_fieldsets(context, request, schema, additional_schemata)
+    try:
+        schema = fti.lookupSchema()
+    except AttributeError:
+        schema = None
```

Comment:
Maybe add a comment here to remind us when we expect this to happen.

Reply:
@jaroel +1 ^^^